app/src/main/kotlin/com/intive/picover/camera/viewmodel/CameraViewModel.kt

```diff
@@ -12,13 +11,9 @@ class CameraViewModel @Inject constructor(
 	imageFileRepository: ImageFileRepository,
 ) : ViewModel() {
 
-	val takenImageUri = mutableStateOf<Uri?>(null)
+	val takenImageUri = imageFileRepository.createTempFileAndGetUri()
```

Comment:
Besides weird usage. Please guys remember that not every get/set/create/store operation needs to be placed in **Repository class. Indeed, it's a common pattern for Android apps but you don't have use it every time.